bskyweb/static.go

```diff
@@ -2,5 +2,5 @@ package bskyweb
 
 import "embed"
 
-//go:embed static/*
+//go:embed all:static/*
```

Comment:
I'm not really familiar with what's going on here, but CI was failing - relevant issue here:

https://github.com/golang/go/issues/43854